tests/admin_scripts/tests.py

```diff
@@ -2459,60 +2451,60 @@ def test_custom_project_template(self):
         The startproject management command is able to use a different project
         template.
         """
-        template_path = os.path.join(custom_templates_dir, "project_template")
+        template_path = custom_templates_dir / "project_template"
         args = ["startproject", "--template", template_path, "customtestproject"]
-        testproject_dir = os.path.join(self.test_dir, "customtestproject")
+        testproject_dir = self.test_dir / "customtestproject"
 
         out, err = self.run_django_admin(args)
         self.assertNoOutput(err)
-        self.assertTrue(os.path.isdir(testproject_dir))
-        self.assertTrue(os.path.exists(os.path.join(testproject_dir, "additional_dir")))
+        self.assertTrue(testproject_dir.is_dir())
+        self.assertTrue((testproject_dir / "additional_dir").exists())
 
     def test_template_dir_with_trailing_slash(self):
         "Ticket 17475: Template dir passed has a trailing path separator"
-        template_path = os.path.join(custom_templates_dir, "project_template" + os.sep)
+        template_path = custom_templates_dir / f"project_template{os.sep}"
```

Comment:
Just a musing: I guess it's one of those things - did this test exist to check support for lazy use of `template_path + "folder"` instead of `os.path.join(template_path, "folder")`, and if so, is it even relevant anymore if we're relying on the behaviour of `pathlib`?